src/main/webapp/resources/js/pages/projects/share/TargetMetadataRestriction.jsx

```diff
@@ -0,0 +1,115 @@
+import { Form, Popover, Select, Space, Tag, Tooltip, Typography } from "antd";
+import React from "react";
+import { useDispatch } from "react-redux";
+import {
+  IconInfoCircle,
+  IconWarningOutlined,
+} from "../../../components/icons/Icons";
+import { blue6, red6 } from "../../../styles/colors";
+import {
+  getColourForRestriction
+} from "../../../utilities/restriction-utilities";
+import { updateMetadataRestriction } from "./shareSlice";
+
+/**
+ * React component to allow the user to select the level of restiction for a
+ * metadata field in the destination project.
+ *
+ * @param {object} field - field to set value on
+ *  project, if it is not in the target project it get the current restriction.
```

Comment:
Change get to `gets`

Reply:
Updated in 72f33d2003edbe3f57e1aa59eb5b344c621378a0